app/views/layouts/user_mailer.html.slim

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+html xmlns="http://www.w3.org/1999/xhtml" 
+  head
+    = stylesheet_link_tag "email/email"
+    meta content="text/html; charset=utf-8" http-equiv="Content-Type"
+    meta content="width=device-width" name="viewport"
+    title
+      = message.subject
+  body
+    table.body data-made-with-foundation=""
+      tr
+        td.center align="center" valign="top" 
+          center data-parsed="" 
+            table.spacer.float-center
+              tbody
+                tr
+                  td height="10px" style="font-size:10px;line-height:10px;"   
```

Comment:
cool, thanks; i would be in favor of adding these utility classes in a future PR